doc/book/connectors/__python.rst

```diff
@@ -39,8 +39,221 @@ For an example of using Python API with
+        -   Yes
+        -   No (last updated in 2016)
+        -   No (last updated in 2018)
+        -   Yes
+
+    *   -   Known Issues
+        -   `issue #18 <https://github.com/igorcoding/asynctnt/issues/18>`__
+        -   `issue #5 <https://github.com/shveenkov/aiotarantool/issues/5>`__
+            (*select()* fails), `issue #16
+            <https://github.com/shveenkov/aiotarantool/issues/16>`__ (Tarantool
+            1.7 is unsupported)
+        -   None
+        -   `issue #105 <https://github.com/tarantool/tarantool-python/issues/105>`__ (Unpack of binary data)
+
+    *   -   Documentation
+        -   Yes (`github.io <https://igorcoding.github.io/asynctnt/>`__)
+        -   No
+        -   No
+        -   Yes (`tarantool.io
+            <https://www.tarantool.io/en/doc/latest/getting_started/getting_started_python/>`__
+            and `readthedocs
+            <https://tarantool-python.readthedocs.io/en/latest/quick-start.en.html>`__
+            (`obsolete
+            <https://github.com/tarantool/tarantool-python/issues/67>`__))
+
+    *   -   Testing / CI / CD
+        -   Github Actions
+        -   No (tests exists)
+        -   No (tests exists)
+        -   Travis CI and Appveyor
+
+    *   -   Github Stars
+        -   51
+        -   38
+        -   17
+        -   78
+
+    *   -   Static Analysis
+        -   Yes (Flake8)
+        -   No
+        -   No
+        -   No
+
+    *   -   Packaging
+        -   `Pip <https://pypi.org/project/asynctnt/>`__
+        -   `Pip <https://pypi.org/project/aiotarantool/>`__
+        -   `Pip <https://pypi.org/project/gtarantool/>`__
+        -   `Deb, RPM, Pip <https://github.com/tarantool/tarantool-python#download-and-install>`__
+
+    *   -   Code coverage
+        -   Yes
+        -   No
+        -   No
+        -   Yes
+
+    *   -   Support asynchronous mode
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes, `asyncio <https://docs.python.org/3/library/asyncio.html>`__
+        -   Yes (`gevent
+            <https://www.gevent.org/api/gevent.event.html#gevent.event.AsyncResult>`__,
+            example: `test_gevent.py
+            <https://github.com/shveenkov/gtarantool/blob/master/tests/test_gevent.py>`__)
+        -   No
+
+    *   -   Batching support
+        -   No
+        -   No
+        -   No
+        -   No (`issue #55 <https://github.com/tarantool/tarantool-python/issues/55>`__)
+
+    *   -   Schema reload, space and index names
+        -   Yes (*auto_refetch_schema*)
+        -   Yes (handle *SchemaReloadException*)
+        -   No
+        -   Yes (handle *SchemaReloadException*)
+
+    *   -   Schema reload, Access tuple fields by names
+        -   Yes
+        -   FIXME
+        -   FIXME
+        -   FIXME
+
+    *   -   `SQL support <https://www.tarantool.io/en/doc/latest/reference/reference_sql/>`__
+        -   Yes (tests/test_op_sql.py)
+        -   No (using call server-side functions)
+        -   No
```

Comment:
Is there a difference between 'No (using call server-side functions)' and 'No'?

Reply:
Updated